angular/src/lib/radio/radio.component.ts

```diff
@@ -19,12 +19,14 @@ const CUSTOM_RADIO_VALUE_ACCESSOR: any = {
   multi: true,
 };
 // tslint:enable:no-use-before-declare
-
 @Component({
   selector: 'md-radio',
   template: `
-    <div class="md-radio-group">
-      <div class="md-input-group md-radio" [ngClass]="wrapperClasses">
+      <div class="md-input-group md-radio"
```

Comment:
Should these classes be the host also?

Reply:
I removed the wrapper div with the class of md-radio-group and moved that to the host. These classes are part of the inner div.